geth/jail/internal/fetch/Makefile

```diff
@@ -1,5 +1,6 @@
-dist-fetch.rice-box.go: dist-fetch/bundle.js
-	rice embed-go
+generate: dist-fetch/bundle.js
+	go generate .
+	gofmt -s -w dist_fetch.go
```

Comment:
Not sure simplify option (`-s`) has any benefit here for autogenerated code.

Reply:
Might not but I don't see any negative impact having `-s`.